src/codeu/chat/server/Controller.java

```diff
@@ -39,6 +41,9 @@ public Controller(Uuid serverId, Model model) {
     this.uuidGenerator = new RandomUuidGenerator(serverId, System.currentTimeMillis());
   }
 
+  public void addMemberToConversation(Uuid conversationID, Uuid owner, Uuid member) {
+    ConversationHeader.members.add(member);
```

Comment:
This is an example of how you get a conversation by Id

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/ff639447ffe6d2e2f46e8781a0ba7be10fbc1c18/src/codeu/chat/server/Controller.java#L165